packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentInformation.gql.js

```diff
@@ -37,19 +37,15 @@ export const GET_PAYMENT_INFORMATION = gql`
     }
     ${AvailablePaymentMethodsFragment}
 `;
-// We disable linting for local fields because there is no way to add them to
-// the fetched schema.
-// https://github.com/apollographql/eslint-plugin-graphql/issues/99
-/* eslint-disable graphql/template-strings */
+
 export const GET_PAYMENT_NONCE = gql`
     query getPaymentNonce($cartId: String!) {
-        cart(cart_id: $cartId) @client {
```

Comment:
In many of the queries we move `@client` to the local such as here for `paymentNonce`. Does this result in many more queries than were previously sent?

Reply:
If the fetchPolicy was `cache-and-network`, I could _maybe_ see this hitting the network now. I was getting a validation error with these on the operation instead of the fields, but now that those directives are merged into the schema, I'm not getting those anymore. I think this can safely be backed out.

Reply:
Reverted these directive changes in b9de346